scala/lambdas/dynamo-formatters/src/test/scala/uk/gov/nationalarchives/dynamoformatters/DynamoFormattersTest.scala

```diff
@@ -308,7 +308,7 @@ class DynamoFormattersTest extends AnyFlatSpec with TableDrivenPropertyChecks wi
       val dynamoTableFields = tableElementNames.toList
       val dynamoTableFieldsMapped = dynamoTableFields.map {
         case "identifiers"           => "id_Test"
-        case "checksumSha256"        => "checksum_sha256"
+        case "checksums"             => "checksum_Algorithm1"
```

Comment:
You can remove this line and then on line 306, you can do this

```
        case File          =>
          val fileDynamoTable = generateFileDynamoTable()
          fileDynamoTable.productElementNames.filterNot(_ == "checksums") ++
          fileDynamoTable.checksums.map(_.algorithm)
```
That way, you can add more checksums to the list in the `generateFileDynamoTable` method meaning the tests will be more realistic

Reply:
That's fine, I'll change it